### ISSUE TITLE
Hey im getting Error 

### ISSUE THREAD

**Issue by SaifulAzam** [mastashake08/laravel-spark-archive#1]
When I Use this comment line "spark install" Im getting Error 
[InvalidArgumentException]  
  Could not find package laravel/spark at any version for your minimum-stability (dev). Check the package spelling or your minimum-stability  

require [--dev] [--prefer-source] [--prefer-dist] [--no-progress] [--no-update] [--no-scripts] [--update-no-dev] [--update-with-dependencies] [--ignore-platform-reqs] [--prefer-stable] [--prefer-lowest] [--sort-packages] [-o|--optimize-autoloader] [-a|--classmap-authoritative] [--] [<packages>]...

  [Symfony\Component\Debug\Exception\FatalThrowableError]  
  Class 'Laravel\Spark\Providers\SparkServiceProvider' not found

**Comment by mastashake08** [comment]
Yes I have to update dependencies I will try and fix this weekend

**Comment by SaifulAzam** [comment]
hey bro did you updated yet?

**Comment by mastashake08** [comment]
Hello!

This is Jyrone Parker, CEO/Founder of Treat Me At Home. Treat Me At Home is a new on-demand app that connects service providers with people who want those services performed from the comfort of their home. Becoming a service provider is 100% free and it always will be. You choose how often you want to work, whether it be full-time or part-time! You can even earn money just by referring others to the app! Whether you would like to use the service or would like to become a service provider I implore you to check out Treat Me At Home New features are being added weekly! Also share the app with your friends and family!

Go Now!: https://treatmeathome.online

If you would like to be removed from the email list please go to https://treatmeathome.online/lead/remove/321
Thank you for your time!

Regards,
Treat Me At Home
